docs/api/util.rst

```diff
@@ -10,6 +10,17 @@ URI Functions
     :members: encode, encode_value, decode, parse_host,
         parse_query_string, unquote_string
 
+ASGI Helpers
+------------
+
+.. _asgi_helpers:
```

Comment:
In the future, I plan to reference this section from other locations in the docs.